symengine/lambda_double.h

```diff
@@ -39,32 +49,39 @@ class LambdaDoubleVisitor : public BaseVisitor<LambdaDoubleVisitor<T>>
 
     T call(const std::vector<T> &vec)
     {
-        return result_(vec);
+        return result_(vec.data());
+    }
+
+    void call(T *outs, const T *inps)
```

Comment:
Cool, so you've changed the signature? - that's great! So LLVM should see all expressions at the same time.